ui/components/app/simulation-preview/amount-pill.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Text } from '../../component-library';
+import {
+  AlignItems,
+  BackgroundColor,
+  BorderRadius,
+  Display,
+  FlexDirection,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import { Numeric } from '../../../../shared/modules/Numeric';
+
+/**
+ * Displays a pill with an amount and a background color indicating whether the amount is an increase or decrease.
+ *
+ * @param props
+ * @param props.isDecrease
+ * @param props.absChange
+ */
+export const AmountPill: React.FC<{
+  isDecrease: boolean;
+  absChange: Numeric;
```

Comment:
For clarity, should we call this `absChangeEth` so it's clear we expect the value to have already been formatted from WEI?